### ISSUE TITLE
Error in the prepopulated relative path new module path

### ISSUE THREAD

**Issue by khoait** [mstuttgart/vscode-odoo-scaffold#1]
When I right-click on a folder, and click New Odoo Module, the prepopulated path is a full path not relative path, for example `/e:/MyProject/custom_addons`. Note the extra forward slash "/" at the beginning of the path. If I continue to use this path, there will be an error.

**Comment by mstuttgart** [comment]
Hi @khoait, you use Vscode in Windows?

**Comment by mstuttgart** [comment]
@khoait I revert commit in 0.1.2 version.

**Comment by captivea-kco** [comment]
I experienced this issue as well.  It seems that fileObj.path returns a path with a leading /, which is not valid on Windows.  Manually removing the slash allows everything to function perfectly.

I was able to resolve it programatically by editing extension.js and adding this **if** statement after relativePath is defined.  It should only impact Windows machines and do nothing on Linux and others.

if (process.platform == 'win32' && relativePath.charAt(0) == '/') {
	relativePath = relativePath.substr(1);
}
